node_modules/oae-core/setpermissions/setpermissions.html

```diff
@@ -5,31 +5,35 @@
 
 <div id="setpermissions-template"><!--
     <div id="setpermissions-form" role="form">
-        <h4>__MSG__VISIBILITY__</h4>
-        <div class="radio">
-            <label>
-                <input type="radio" value="private" name="setpermissions-visibility" class="pull-left" {if visibility === 'private'} checked="checked"{/if}/>
-                <i class="fa fa-oae-private pull-left"></i>
-                <span>__MSG__PRIVATE__</span>
-            </label>
-        </div>
-        <div class="radio">
-            <label>
-                <input type="radio" value="loggedin" name="setpermissions-visibility" class="pull-left" {if visibility === 'loggedin'} checked="checked"{/if}/>
-                <i class="fa fa-oae-loggedin pull-left"></i>
-                <span>${oae.data.me.tenant.displayName|encodeForHTML}</span>
-            </label>
-        </div>
-        <div class="radio">
-            <label>
-                <input type="radio" value="public" name="setpermissions-visibility" class="pull-left" {if visibility === 'public'} checked="checked"{/if}/>
-                <i class="fa fa-oae-public pull-left"></i>
-                <span>__MSG__PUBLIC__</span>
-            </label>
-        </div>
+        <fieldset>
```

Comment:
Yes they do. (I missed those because I was grepping for the bootstrap class instead of the input type.) Alas, they're going to be a bit more challenging.